doc: update todo app documentation (#188)
couchbaselabs/mobile-training-todo: README.md

# ToDo Application
ToDo iOS sample app built with [CouchbaseLite 2.5.0](https://github.com/couchbase/couchbase-lite-ios/tree/release/iridium). 
The application has Android, DotNet, Objective-C and Swift versions.

## Requirements

> - XCode 8.3

Author: ```suggestion
- XCode 10.0
```

## How to build and run?

> 1. Clone the project and checkout `release/iridium` branch.

Author: ```suggestion
1. Clone the project and checkout `feature/2.5` branch.
```

 ```
 $ git clone https://github.com/couchbaselabs/mobile-training-todo.git
 $ git checkout feature/2.5
 ```
 

> 2. cd into the project, `android`, `dotnet`, `objc` or `swift`.

Author: ```suggestion
2. cd into the project: `android`, `dotnet`, `objc` or `swift`.
```
